test/inspector/test-inspector-ip-detection.js

```diff
@@ -22,7 +22,7 @@ function checkListResponse(instance, err, response) {
   const match = wsUrl.match(/^ws:\/\/(.*):9229\/(.*)/);
   assert.strictEqual(ip, match[1]);
   assert.strictEqual(res['id'], match[2]);
-  assert.strictEqual(ip, res['devtoolsFrontendUrl'].match(/.*ws=(.*):9229/)[1]);
+  assert.strictEqual(ip, res['devtoolsFrontendUrl'].match(/ws=(.*):9229/)[1]);
```

Comment:
This change actually can have an effect on the match returned:

```js
const string = 'ws=0.0.0.0:9299 ws=1.1.1.1:9299';

string.match(/.*ws=(.*):9229/)[1] // => '1.1.1.1'

string.match(/ws=(.*):9229/)[1] // => 0.0.0.0:9299 ws=1.1.1.1
```

I'm not sure whether this matters for this particular test case.